server/index.js

```diff
@@ -386,8 +386,28 @@ app.delete('/api/users/user-events', exjwt({ secret: 'secret' }), wrap(async (re
   res.json(events);
 }));
 
+app.get('/api/users/user-ratings', exjwt({ secret: 'secret' }), wrap(async (req, res) => {
+  const userId = req.user.id;
+  const user = await db.User.findById(userId);
+  const ratings = await user.getRatings();
+  res.json(ratings);
+}));
+
+app.post('/api/users/user-ratings', exjwt({ secret: 'secret' }), wrap(async (req, res) => {
+  const userId = req.user.id;
+  const { informed, titleBias, articleBias, sourceTrust, articleId } = req.body;
+
+  const user = await db.User.findById(userId);
+  const article = await db.Article.findById(articleId);
+  const rating = await db.Rating.create({ informed, titleBias, articleBias, sourceTrust });
+
+  await article.addRating(rating);
+  await user.addRating(rating);
+  res.json(user);
+}));
+
 // auth
-app.get('/api/auth/login', wrap(async (req, res) => {
+app.get('/api/auth/login', exjwt({ secret: 'secret' }), wrap(async (req, res) => {
```

Comment:
was this fixed in a different PR? i know this was causing errors, should not have the exjwt function in there
@michellelockett